src/jwt/decode.ts

```diff
@@ -0,0 +1,14 @@
+import { decodeFromBase64Url } from "./utils";
+import type { JWTRegisteredClaims } from "./types";
+
+export function decodeJWT<T extends Record<string, any> = Record<string, any>>(
```

Comment:
Please prefer `unknown` for type safety 
```suggestion
export function decodeJWT<T extends Record<string, unknown> = Record<string, unknown>>(
```